src/GitHub.VisualStudio/Menus/CopyLink.cs

```diff
@@ -45,7 +48,8 @@ public void Activate([AllowNull]object data = null)
 
         public bool CanShow()
         {
-            return IsGitHubRepo();
+            return IsGitHubRepo().Wait(250); 
```

Comment:
Ah, I see what you're trying to do there, but that's not going to return quite the result you expect if it's not a github repo... Consider:

- User has repo with remote set to `http://a-server-that-isn-t-github/user/repo`
- First time `IsGitHubRepo` is called and detects it's not `github.com`, so it checks if this server has been probed already. It hasn't, so it checks if the server returns a github header and caches the result, which is `false`
- 250ms have elapsed before the check returns, so `Wait` returns false and the menu is hidden
- Second time `IsGitHubRepo` is called and detects it's not `github.com`, so it checks if this server has been probed already. It has, so it returns the probe result, which is `false`.
- 250ms have not elapsed, so `Wait` returns `true` and the menu is shown. Ooops!